app/helpers/application_helper.rb

```diff
@@ -642,4 +642,31 @@ def list_to_string(list)
   end
 
 
+  def t(*args)
+    translation = I18n.t(*args).downcase
+    term = I18n.t("resource_term.ontology")
+    plural_term = I18n.t("resource_term.ontology_plural")
+    single_term = I18n.t("resource_term.ontology_single")
+    resource = I18n.t("resource_term.#{$RESOURCE_TERM}")
```

Comment:
this variable need to be added to configuration file `bioportal_config_env.rb.sample`